src/modules/media/services/media.service.spec.ts

```diff
@@ -109,31 +110,34 @@ describe('MediaService', () => {
 	});
 
 	describe('findById', () => {
-		it('returns the es response from the main _search endpoint if no index is specified', async () => {
-			nock('http://elasticsearch/').post('/_search').reply(201, getMockMediaResponse());
-			const response = await mediaService.findById('123');
-			expect(response.hits.total.value).toBe(2);
-			expect(response.hits.hits.length).toBe(2);
+		it('returns the full object details as retrieved from the DB', async () => {
+			mockDataService.execute.mockResolvedValueOnce(objectIe);
+			const response = await mediaService.findById('8911p09j1g');
```

Comment:
replace all hard coded values '8911p09j1g' with 
```
objectIe.data.object_ie_by_pk.schema_identifier
```

So if the mock changes, the test doesn't break.